server/src/test/java/org/elasticsearch/cli/MultiCommandTests.java

```diff
@@ -221,7 +221,7 @@ public void testErrorDisplayedWithDefault() throws Exception {
         multiCommand.subcommands.put("throw", new ErrorThrowingSubCommand());
         executeMain("throw", "--silent");
         assertThat(terminal.getOutput(), is(emptyString()));
-        assertThat(terminal.getErrorOutput(), equalTo("ERROR: Dummy error\n"));
+        assertThat(terminal.getErrorOutput(), equalTo("ERROR: Dummy error" + System.lineSeparator()));
```

Comment:
We could probably just remove the line ending altogether and use startsWith? This test is trying to assert errors are printed when --silent is used, not check anything about terminal output.

Reply:
Good call. Thanks!